### ISSUE TITLE
data processing script in code inconsistent with paper

### ISSUE THREAD

**Issue by truetone2022** [MCZhi/DTPP#4]
![image](https://github.com/MCZhi/DTPP/assets/45789117/1d5bf0c2-7861-414a-87e0-1f8d4f6d65f8)
![image](https://github.com/MCZhi/DTPP/assets/45789117/f07bd32f-e0c9-476a-9425-01aa622bcab4)
![image](https://github.com/MCZhi/DTPP/assets/45789117/fe707c69-e0e7-44a2-945c-82c4bdb87e17)

**Comment by truetone2022** [comment]
The nuplan valset subset used in the paper is used as the training set, and the subset of the nuplan test set is used as the test set.
The code only processes the nuplan miniset, and does not mention how to extract the training data and test data that strictly correspond to the paper.

**Comment by MCZhi** [comment]
The parameters shown in the code are just an example. You can use any subset of data for your research. Just replace "mini" with "val" or "test".

**Comment by truetone2022** [comment]
Are the 100k scenarios in the training set randomly selected? How can I get the 200 scenarios in the test set that are exactly the same as those in your paper?

**Comment by MCZhi** [comment]
Yes. They are randomly selected. We will provide the trained model and testing scenarios later.

**Comment by truetone2022** [comment]
Could you share the test set containing 200 scenarios or the corresponding test set processing script earlier?

**Comment by truetone2022** [comment]
Are the 100k scenarios in the training set randomly selected according to the num_scenarios_per_type limit of 10k scenarios per type.

**Comment by truetone2022** [comment]
Are the 100k scenarios in the training set randomly selected according to the num_scenarios_per_type limit of 10k scenarios per type?

**Comment by MCZhi** [comment]
We have updated the data processing code and uploaded a base pre-trained model and test set.
